.github/workflows/release.yaml

```diff
@@ -0,0 +1,121 @@
+name: "Release"
+run-name: Release ${{ inputs.tag }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      revision:
+        type: string
+        description: "commit sha1 revision where to start the release, must be the long commit sha1"
+        required: true
+      tag:
+        type: string
+        description: "the new tag/release version to create"
+        required: true
+
+env:
+  RELEASE_BRANCH: feature/release-${{inputs.tag}}
+
+jobs:
+  prepare-version:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.ACTIONS_APP_ID }}
+          private-key: ${{ secrets.ACTIONS_APP_PRIVATE_KEY }}
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{inputs.revision}}
+          token: ${{ steps.app-token.outputs.token }}
+      - name: Install semver tool
+        run: |
+          curl --fail -LO https://raw.githubusercontent.com/fsaintjacques/semver-tool/3.3.0/src/semver
+          chmod +x ./semver
+      - name: Check hotfix releases
+        env:
+          SEMVER_TAG: ${{ inputs.tag }}
+        run: |
+          if [[ "$SEMVER_TAG" =~ ^([[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+)\.([[:digit:]]+)(.*) ]]; then
+            echo "Hotfix version detected: ${BASH_REMATCH[1]}, hotfix: ${BASH_REMATCH[2]}, prerelease: ${BASH_REMATCH[3]}"
+            echo HOTFIX_VERSION="${BASH_REMATCH[2]}" >> $GITHUB_ENV
+            echo NEW_VERSION_SUFFIX=".${BASH_REMATCH[2]}${BASH_REMATCH[3]}" >> $GITHUB_ENV
+
+            # if a hotfix version is detected, we 'tweak' the tag to make it a valid semver tag
+            # This will only be usd for internal validation, the "official" tag will be the 4-digit
+            # tag that was provided
+            SEMVER_TAG="${BASH_REMATCH[1]}-hotfix.${BASH_REMATCH[2]}${BASH_REMATCH[3]}"
+          fi
```

Comment:
No, there is no HOTFIX at all in MetalK8s

Reply:
Removed!